atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SweepCommand.java

```diff
@@ -98,11 +98,16 @@ public int execute(final AtlasDbServices services) {
         Map<TableReference, Optional<byte[]>> tableToStartRow = Maps.newHashMap();
 
         if ((table != null)) {
-            Optional<byte[]> startRow = Optional.of(new byte[0]);
-            if (row != null) {
-                startRow = Optional.of(decodeStartRow(row));
+            TableReference tableToSweep = TableReference.createUnsafe(table);
+            if (services.getKeyValueService().getAllTableNames().contains(tableToSweep)) {
+                Optional<byte[]> startRow = Optional.of(new byte[0]);
+                if (row != null) {
+                    startRow = Optional.of(decodeStartRow(row));
+                }
+                tableToStartRow.put(tableToSweep, startRow);
+            } else {
+                printer.info("The table passed in to sweep {} does not exist", tableToSweep);
```

Comment:
Also, it would be better to invert the if (to deal with the error case first) and then return 1, as above.